examples/imagenet/train.py

```diff
@@ -217,16 +220,11 @@ def save_checkpoint(state, workdir):
 
 def sync_batch_stats(state):
   """Sync the batch statistics across replicas."""
-
-  # Each device has its own version of the running average batch statistics and
```

Comment:
Do we no longer need this code?

Reply:
the new code looks good. I would not delete the comment because users often don't understand why this sync happens